hack/ci/e2e.sh

```diff
@@ -187,9 +187,30 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
-    # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    # ginkgo regexes, use CI k/k presubmit regex by default
+    SKIP="${SKIP:-"\\[Slow\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+    FOCUS="${FOCUS:-""}"
+
+    # skip known issues
+    # [sig-node] RuntimeClass should run a Pod requesting a RuntimeClass with a configured handler [NodeFeature:RuntimeHandler]
+    # https://github.com/kubernetes/kubernetes/issues/80098
+    SKIP="RuntimeClass.should.run.a.Pod.requesting.a.RuntimeClass.with.a.configured.handler|${SKIP}"
+    # [sig-storage] In-tree Volumes [Driver: nfs]
+    # [sig-storage] PersistentVolumes NFS
+    # TODO: kind nodes can't mount nfs volumes: install apt install -y nfs-common in the base image, ...??
+    SKIP="In-tree.Volumes.\\[Driver:.nfs\\]|PersistentVolumes.NFS|${SKIP}"
+    # [sig-storage] PersistentVolumes-local
+    # [sig-storage] In-tree Volumes [Driver: local]
+    # fails because sh: 1: sudo: not found, TODO check if we can detect sudo test/e2e/storage/utils/local.go:
+    SKIP="PersistentVolumes-local|In-tree.Volumes.\\[Driver:.local\\]|${SKIP}"
+    # [sig-auth] PodSecurityPolicy
+    SKIP="PodSecurityPolicy|${SKIP}"
+    # [sig-network] Network should set TCP CLOSE_WAIT timeout
+    # Use ssh to login to the kind nodes TODO: change to docker exec commands, install ssh server, ...
+    SKIP="Network.should.set.TCP.CLOSE_WAIT.timeout|${SKIP}"
+    # [sig-cli] Kubectl client [k8s.io] Simple pod should support exec through an HTTP proxy 
+    SKIP="Simple.pod.should.support.exec.through.an.HTTP.proxy|${SKIP}"
```

Comment:
is this one using SSH?

Reply:
nope, it seems is not able to use the proxy 🤷‍♂ 
> Jul 12 04:47:56.459: INFO: Missing expected log result on proxy server for https_proxy. Expected: "Accepting CONNECT to 127.0.0.1:35545", got ""

Reply:
is that because it's tying to use the "External" address for the api server?

Reply:
... that's a bit of a problem for kind :thinking:

Reply:
good catch, it´s because is using the api server endpoint on localhost.
Passing the host as parameter to the e2e tests with the node API server address works
` bazel-bin/test/e2e/e2e.test -ginkgo.focus="Simple pod should support exec through an HTTP proxy" -host https://172.17.0.5:6443`

Reply:
hmmm I wonder if this works on macbook ... the usual suspect :P

Reply:
I'm running them on Linux